quic/s2n-quic-core/src/recovery/hybrid_slow_start.rs

```diff
@@ -17,25 +17,55 @@ pub struct HybridSlowStart {
     pub(super) threshold: f32,
     max_datagram_size: u16,
     rtt_round_end_time: Option<Timestamp>,
+    use_hystart_plus_plus: bool,
+    ss_growth_divisor: f32,
+    css_count: usize,
+    css_baseline_min_rtt: Duration,
+    css_threshold: f32,
 }
 
 /// Minimum slow start threshold in multiples of the max_datagram_size.
 /// Defined as "hystart_low_window" in tcp_cubic.c
 const LOW_SSTHRESH: u16 = 16;
+/// Factor for dividing the RTT to determine the threshold. Defined in tcp_cubic.c (not a constant)
+//= https://tools.ietf.org/id/draft-ietf-tcpm-hystartplusplus-04.txt#section-4.2
+//#   o  RttThresh = clamp(MIN_RTT_THRESH, lastRoundMinRTT / 8, MAX_RTT_THRESH)
+const THRESHOLD_DIVIDEND: u32 = 8;
+
+//= https://tools.ietf.org/id/draft-ietf-tcpm-hystartplusplus-04.txt#section-4.3
+//# It is RECOMMENDED that a HyStart++ implementation use the following
+//# constants:
+//#
+//# *  MIN_RTT_THRESH = 4 msec
+//#
+//# *  MAX_RTT_THRESH = 16 msec
+//#
+//# *  N_RTT_SAMPLE = 8
+//#
+//# *  CSS_GROWTH_DIVISOR = 4
+//#
+//# *  CSS_ROUNDS = 5
+
 /// Number of samples required before determining the slow start threshold.
 /// Defined as "HYSTART_MIN_SAMPLES" in tcp_cubic.c
 const N_SAMPLING: usize = 8;
 /// Minimum increase in delay to consider. Defined as"HYSTART_DELAY_MIN" in tcp_cubic.c
 const MIN_DELAY_THRESHOLD: Duration = Duration::from_millis(4);
 /// Maximum increase in delay to consider. Defined as"HYSTART_DELAY_MAX" in tcp_cubic.c
 const MAX_DELAY_THRESHOLD: Duration = Duration::from_millis(16);
-/// Factor for dividing the RTT to determine the threshold. Defined in tcp_cubic.c (not a constant)
-const THRESHOLD_DIVIDEND: u32 = 8;
+/// Growth devisor for CSS phase
+const CSS_GROWTH_DIVISOR: f32 = 4.0;
+/// Maximum rounds for CSS phase
+const CSS_ROUNDS: usize = 5;
+/// environment variable for using hystart++
+#[cfg(feature = "std")]
+const USE_HYSTART_PLUS_PLUS: &str = "S2N_UNSTABLE_USE_HYSTART_PP";
 
 impl HybridSlowStart {
     /// Constructs a new `HybridSlowStart`. `max_datagram_size` is used for determining
     /// the minimum slow start threshold.
     pub fn new(max_datagram_size: u16) -> Self {
+        let _hystart_param = Self::get_hystart_parameter();
```

Comment:
why the underscore? I don't think you even need a variable, could you just call it directly below like: `use_hystart_plus_plus: Self::get_hystart_parameter(),`

Reply:
ok. updated.